src/zos_files/zowe/zos_files_for_zowe_sdk/datasets.py

```diff
@@ -538,18 +626,32 @@ def download_binary(self, dataset_name, output_file, with_prefixes=False):
 
         Parameters
         ----------
-        dataset_name:str - Name of the dataset to download
-        output_file:str - Name of the local file to create
-        with_prefixes:boolean - If true, include a four big endian bytes record length prefix.
-                                The default is False
+        dataset_name:str
+            Name of the dataset to download
+        output_file:str
+            Name of the local file to create
+        with_prefixes:boolean
+            If true, include a four big endian bytes record length prefix. The default is False
```

Comment:
Should we use formatting consistent with `get_binary_content`?
```suggestion
            If true, include a four big endian bytes record length prefix. Default: False
```